docs/api_openapi_specification.yml

```
@@ -60,28 +60,28 @@ paths:
               $ref: '#/components/headers/GovukApiUserWriteRateLimitRemaining'
             Govuk-Api-User-Write-RateLimit-Reset:
               $ref: '#/components/headers/GovukApiUserWriteRateLimitReset'
-            Govuk-Client-Device-Id-Write-RateLimit-Limit:
-              $ref: '#/components/headers/GovukClientDeviceIdWriteRateLimitLimit'
-            Govuk-Client-Device-Id-Write-RateLimit-Remaining:
-              $ref: '#/components/headers/GovukClientDeviceIdWriteRateLimitRemaining'
-            Govuk-Client-Device-Id-Write-RateLimit-Reset:
-              $ref: '#/components/headers/GovukClientDeviceIdWriteRateLimitReset'
+            Govuk-Client-User-Id-Write-RateLimit-Limit:
+              $ref: '#/components/headers/GovukClientUserIdWriteRateLimitLimit'
+            Govuk-Client-User-Id-Write-RateLimit-Remaining:
+              $ref: '#/components/headers/GovukClientUserIdWriteRateLimitRemaining'
+            Govuk-Client-User-Id-Write-RateLimit-Reset:
+              $ref: '#/components/headers/GovukClientUserIdWriteRateLimitReset'
         "429":
           description: |
-            Too many requests to write endpoints from an Api User or Device ID
+            Too many requests to write endpoints from an Api User or User ID
```

```suggestion
            Too many requests to write endpoints from an API User or Client User ID
```

Just so we have a qualifier before using "User"

(and apologies I know you didn't introduce the API bit)